macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -430,29 +431,88 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
             return Application.appDelegate.windowControllersManager
         })
 
-        self.isAuthV2Enabled = featureFlagger.isFeatureOn(.privacyProAuthV2)
```

Comment:
From here to line 515 is just the usual subscription init swapped around V1<>V2 and the new `AuthMigrator `